python/ct/crypto/tools/verify_util.py

```diff
@@ -13,7 +13,7 @@
 
   For example:
 
-  verify_util.py verify_sct --sct=cert_sct.tls --log_key=log_key.pem cert.pem
+  verify_util.py verify_sct --sct=cert_sct.tls --log_key=log_key.pem cert.pem [cert.pem...]
```

Comment:
This chain should not be necessary - a single PEM file can contain multiple certificates and that's the convention around the codebase.